src/server.c

```diff
@@ -1335,18 +1335,24 @@ int serverCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
                 used = kvstoreSize(server.db[j].keys);
                 vkeys = kvstoreSize(server.db[j].expires);
                 if (used || vkeys) {
-                    serverLog(LL_VERBOSE, "DB %d: %lld keys (%lld volatile) in %lld slots HT.", j, used, vkeys, size);
+                    serverLog(LL_VERBOSE, "DB %d: %lld keys (%lld volatile) in %lld slots HT.", j, used,
+                              vkeys, size);
                 }
             }
         }
     }
 
     /* Show information about connected clients */
     if (!server.sentinel_mode) {
+        char hmem[64];
+        size_t zmalloc_used = zmalloc_used_memory();
```

Comment:
why not move the get and convert to human to be in the scope  `run_with_period(5000) {...}`?

Reply:
@zvi-code Thank you for your feedback.

Reply:
@madolson please also take a look.